hack/lib/version.sh

```diff
@@ -159,7 +159,7 @@ kube::version::ldflags() {
     ldflags+=(
       "-X '${KUBE_GO_PACKAGE}/pkg/version.${key}=${val}'"
       "-X '${KUBE_GO_PACKAGE}/vendor/k8s.io/client-go/pkg/version.${key}=${val}'"
-      "-X '${KUBE_GO_PACKAGE}/pkg/kubectl/version.${key}=${val}'"
+      "-X '${KUBE_GO_PACKAGE}/staging/src/k8s.io/kubectl/pkg/version.${key}=${val}'"
```

Comment:
why does this use staging and client-go uses vendor? did you verify `kubectl version` returns the correct output?

Reply:
You were correct; the vendor version is what works. I also updated the `def.bzl` to use a `vendor` path. I validated using the following:

`make clean; make WHAT=cmd/kubectl`

then checking the resulting binary with:

`_output/local/bin/linux/amd64/kubectl version --client`

which showed the version had correctly been recorded.

```
Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.0-alpha.1.340+79d153d8d157d6-dirty", GitCommit:"79d153d8d157d67c1cee5c1002f4e5d79934d299", GitTreeState:"dirty", BuildDate:"2019-07-26T22:39:49Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
```

I also checked the bazel output with:

`bazel clean; bazel build //cmd/kubectl`

checking the resulting output at:

`bazel-bin/cmd/kubectl/linux_amd64_pure_stripped/kubectl version --client`

which showed:

```
Client Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.0-alpha.1.340+79d153d8d157d6-dirty", GitCommit:"79d153d8d157d67c1cee5c1002f4e5d79934d299", GitTreeState:"dirty", BuildDate:"2019-07-26T22:45:49Z", GoVersion:"go1.12.7", Compiler:"gc", Platform:"linux/amd64"}
```